### PR TITLE
:bar_chart: Migrate grapher charts to Data Pages (PabloA)

### PR DESCRIPTION
**Pull request by Marigold**
I've taken all grapher charts from our [data pages candidate list](https://docs.google.com/spreadsheets/d/1SyyRR_iaokoc-tkDVQ24F4gW7cf47mLRdEyr8ZSBYo0/edit#gid=0) and ran `etl-metadata-migrate` to pre-generate YAML files. This should hopefully ease the migration. Generated YAML files contain all metadata information we have about indicator (including everything from garden step). This means we might end up with duplicate information in garden & grapher YAML files. It's up to you whether you decide to refactor it and move everything to `garden` channel. I don't think it's necessary now (maybe with next update).

## Instructions

Work on this branch and commit back to it. For all data pages from the list:

1. Look up their YAML file from this PR
2. Fill in all TBD fields (update or remove), remove `description` field
3. Push new commit which will trigger **ETL rebuild of staging server**
4. Wait until ETL is rebuilt
5. Check your data page at http://staging-site-pabloa-data-pages/admin/datapage-preview/[variable_id] - you can get the link from the column "Data Page Preview" in the [spreadsheet](https://docs.google.com/spreadsheets/d/1SyyRR_iaokoc-tkDVQ24F4gW7cf47mLRdEyr8ZSBYo0/edit#gid=0)

Alternatively, you can run it locally with
```
STAGING=pabloa-data-pages etl ... --grapher
```

### CODE REVIEW

**Comment by Marigold** [comment]
@paarriagadap it's all yours! Let me know if something is not clear or if you have any ideas for improvement. By the way @veronikasamborska1994 is experimenting with auto-filling these fields with ChatGPT, so you may coordinate with her if you think ChatGPT will do a good job.

**Comment by paarriagadap** [comment]
Amazing @Marigold, thanks! I will start looking at it today. And how is ChatGPT working for you, @veronikasamborska1994? I will try without it first, as many things are pretty standardized.

**Comment by veronikasamborska1994** [comment]
@paarriagadap hey! I've written a couple of scripts that I think are going to be pretty useful (though some manual curation will probably always be necessary). I am going to try to integrate with the etl today but if you want to play around with them sooner let me know and I can send them over!

File: etl/steps/data/grapher/ggdc/2020-10-01/ggdc_maddison.meta.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+definitions:
```
Comment (paarriagadap) [inline etl/steps/data/grapher/ggdc/2020-10-01/ggdc_maddison.meta.yml:1]: @Marigold Should we add origins for this one or is it ok that it works with sources for now?

Comment (Marigold) [reply etl/steps/data/grapher/ggdc/2020-10-01/ggdc_maddison.meta.yml:1]: This one already uses the origin propagated from the snapshot.

**Comment by paarriagadap** [comment]
Thanks @Marigold! I will merge this commenting out most of V-Dem's indicators, as discussed [here](https://github.com/owid/owid-issues/issues/1209#issuecomment-1842432417).

**Pull request merged**